.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+* @brave/crypto-wallets-core @brave/sec-team
```

Comment:
Looks like the repo needs to be updated so that each of these teams have write access to the library. The syntax here is good as it is and matches how we've done this in other repos from what I can see.